app/src/main/java/com/odysee/app/adapter/ClaimListAdapter.java

```diff
@@ -790,6 +814,13 @@ private void toggleSelectedClaim(Claim claim) {
         notifyItemChanged(items.indexOf(claim));
     }
 
+    private long loadLastPlaybackPosition(Claim claim) {
+        String url = !Helper.isNullOrEmpty(claim.getShortUrl()) ? claim.getShortUrl() : claim.getPermanentUrl();
+        String key = String.format("PlayPos_%s", url.replaceAll("#", ":") /* Use new LBRY URL format */);
```

Comment:
You should use `LbryUri.normalize(url)` here.

The SDK was supposed to have been updated to return ':' instead of '#' for all URLs, but it seems that hasn't been implemented yet.